samples/snippets/src/main/java/com/example/spanner/HubbleTransactionsCodeLab.java

```diff
@@ -34,31 +37,40 @@ public void createInterleaved(DatabaseAdminClient dbAdminClient, DatabaseId id)
         id,
         Arrays.asList(
             "CREATE TABLE Mailbox("
-                + " sid INT64 NOT NULL,"
-                + " state STRING(MAX),"
+                + "sid INT64 NOT NULL,"
+                + "guid STRING(MAX),"
```

Comment:
Why do we need this?

Reply:
You are right I had kept it for different logic but don't need it.